lib/rubocop/eightyfourcodes/version.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module EightyFourCodes
+    VERSION = '2.0.0'
```

Comment:
Should this match the changelog?

```suggestion
    VERSION = '0.0.3'
```